### PR TITLE
MDT- Update Battery Alert Conditions and Battery Page Content

### PR DESCRIPTION
**Pull request by maharielrosario**
## Description
Update Battery Alert Conditions and Battery Page Content

## Testing done
Unit tests

## Screenshots
screenshot-localhost_3001-2020 05 21-15_25_00

screenshot-localhost_3001-2020 05 21-15_25_00 (1)

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by bradl3yC** [comment]
@mr0sari0 I'm confused where eligibility is coming from other than as a default prop or mocked in a unit test - is the API passing this back to us?

**Comment by mr0sari0** [comment]
> @mr0sari0 I'm confused where eligibility is coming from other than as a default prop or mocked in a unit test - is the API passing this back to us?

Yes @camerontesterman created this for us in the API.

File: src/applications/disability-benefits/2346/components/Batteries.jsx [hunk 1]
```diff
@@ -35,41 +35,29 @@ class Batteries extends Component {
...
-    const areBatterySuppliesIneligible = batterySupplies.every(
-      batterySupply => batterySupply.availableForReorder === false,
-    );
+    const areBatterySuppliesEligible = eligibility.batteries;
+    const haveBatteriesBeenOrderedInLastFiveMonths =
+      batterySupplies.length > 0
+        ? batterySupplies.every(
+            battery => currentDate.diff(battery.lastOrderDate, 'months') <= 5,
+          )
+        : false;
```
Comment (bradl3yC) [inline src/applications/disability-benefits/2346/components/Batteries.jsx:49]: You can actually rewrite this to 
```
batterySupplies.every(battery => currentDate.diff(battery.lastOrderDate, 'months') <= 5).length > 0
``` 
to remove the ternary

**Review by bradl3yC — APPROVED** [review]
LGTM just refactor that one condition

File: src/applications/disability-benefits/2346/components/Batteries.jsx (hunk 1 above)
Comment (mr0sari0) [reply src/applications/disability-benefits/2346/components/Batteries.jsx:49]: > You can actually rewrite this to
> 
> ```
> batterySupplies.every(battery => currentDate.diff(battery.lastOrderDate, 'months') <= 5).length > 0
> ```
> 
> to remove the ternary

On user 1, i'm getting `false` for its value when it should be `true`. The `every` function is returning `true` as it should.  I got rid of the length comparison to 0 and just left length and got `undefined`.  I think it's trying to compare the length of a boolean which is why it comes out `undefined`.

Comment (bradl3yC) [reply src/applications/disability-benefits/2346/components/Batteries.jsx:49]: Still, you can run array prototype methods on an empty array - you shoudln't need a length comparison and thus can remove the ternary

Comment (mr0sari0) [reply src/applications/disability-benefits/2346/components/Batteries.jsx:49]: I could add a check prior to `every`. Something like:

`batterySupplies.length > 0 &&
      batterySupplies.every(
        battery => currentDate.diff(battery.lastOrderDate, 'months') <= 5,
      );`

**Comment by mr0sari0** [comment]
> LGTM just refactor that one condition

Cool, I just pushed an update.

**Pull request merged**